pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -212,6 +213,7 @@ func NewLegacyRegistry() *LegacyRegistry {
 			NodeAffinityPriority:        1,
 			TaintTolerationPriority:     1,
 			ImageLocalityPriority:       1,
+			EvenPodsSpreadPriority:      2,
```

Comment:
can you describe what this addition is doing? if I'm reading correctly, the priority used to be set to `1` below... if so, why is this changing?

Reply:
It's a leftover of #91258 - which missed the legacy Policy path.

Do you prefer to cover this in a separate PR?

Reply:
lets do this in a separate small PR that references the original issue.

Reply:
SG, let me revert the change in this PR.